e2e_deployment_files/edgehub_restart_deployment.template.json

```diff
@@ -0,0 +1,230 @@
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "IOT_HUB_CONNECTION_STRING": {
+                "value": "<IoTHubConnectionString>"
+              },
+              "directMethodEnable": {
+                "value": "true"
+              },
+              "directMethodTargetModuleId": {
+                "value": "directMethodReceiver1"
+              },
+              "messageEnable": {
+                "value": "true"
+              },
+              "reportingEndpointUrl": {
+                "value": "http://localhost:5001"
+              },
+              "restartIntervalInMins": {
+                "value": "5"
+              },
+              "sdkRetryTimeoutMilisec": {
+                "value": "20"
+              },
+              "testDuration": {
+                "value": "00:00:10"
+              },
+              "testStartDelay": {
+                "value": "00:00:02"
+              },
+              "trackingId": {
+                "value": "<TrackingId>"
+              }
+            },
+            "settings": {
+              "image": "<Container_Registry>/microsoft/azureiotedge-edgehub-restart-tester:<Build.BuildNumber>-linux-<Architecture>",
+              "createOptions": "{\"HostConfig\":{\"NetworkMode\":\"host\"},\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}}}"
+            }
+          },
+          "relayer1": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "transportType": {
+                "value": "Amqp"
+              },
+              "inputName": {
+                "value": "input1"
+              },
+              "outputName": {
+                "value": "output1"
+              },
+              "testResultCoordinatorUrl": {
+                "value": "http://testResultCoordinator:5001"
+              }
+            },
+            "settings": {
+              "image": "<Container_Registry>/microsoft/azureiotedge-relayer:<Build.BuildNumber>-linux-<Architecture>",
+              "createOptions": ""
+            }
+          },
+          "testResultCoordinator": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "Logging:LogLevel:Microsoft": {
+                "value": "Error"
+              },
+              "trackingId": {
+                "value": "<TrackingId>"
+              },
+              "testStartDelay": {
+                "value": "<TestStartDelay>"
+              },
+              "testDuration": {
+                "value": "<TestDuration>"
+              },
+              "verificationDelay": {
+                "value": "<TestResultCoordinator.VerificationDelay>"
+              },
+              "eventHubConnectionString": {
```

Comment:
I think some parameters are not required for edgehub restart scenario; but I think it is ok to leave it now and update it later if we want.

Reply:
ok